packages/manager/modules/pci/src/projects/new/config/controller.js

```diff
@@ -18,26 +19,61 @@ export default class PciProjectNewConfigCtrl {
     }).join(', ');
   }
 
-  /* -----  End of Helpers  ------ */
-
-  /* =============================
-  =            Events            =
-  ============================== */
-
-  onProjectConfigFormSubmit() {
-    let setDescrPromise = Promise.resolve(true);
-
+  setCartProjectItem() {
     if (
       this.model.description &&
       !this.cart.projectItem.descriptionConfiguration
     ) {
-      setDescrPromise = this.pciProjectNew.setCartProjectItemDescription(
+      return this.pciProjectNew.setCartProjectItemDescription(
         this.cart,
         this.model.description,
       );
     }
 
-    return setDescrPromise.then(() => this.goToPayment());
+    return Promise.resolve();
+  }
+
+  setHdsOptionItem() {
+    const { cartId, projectItem, hdsItem } = this.cart;
+    if (this.model.hds && !hdsItem) {
+      const priceMode = this.hds.option.prices.find(({ capacities }) =>
+        capacities.includes('renew'),
+      );
+      return this.orderCart
+        .addNewOptionToProject(cartId, {
+          duration: priceMode.duration,
+          itemId: projectItem.itemId, // Add HDS option to project
+          planCode: this.hds.option.planCode,
+          pricingMode: priceMode.pricingMode,
+          quantity: 1,
+        })
+        .then((option) => this.cart.addItem(option));
+    }
+    if (!this.model.hds && hdsItem) {
+      return this.orderCart
+        .deleteItem(cartId, hdsItem.itemId)
+        .then(() => this.cart.deleteItem(hdsItem));
+    }
+    return Promise.resolve();
+  }
+
+  sendTracking() {
+    this.trackClick('PublicCloud::pci::projects::new_project_continue');
```

Comment:
in public cloud app 'PublicCloud' is configured to be a prefix with atInternetConfigurationProvider, i'm not sure that the tracking id is good here

Reply:
Thanks @floatovh.
From my personal test, based on staging it works fine.
But I asked someone from tracking team to be sure is working like they expect.
I am waiting response.

Reply:
So i got a response, for tracking team it works fine.
No issue, thanks @frenautvh.